.github/ISSUE_TEMPLATE/improve-existing-docs.yaml

```diff
@@ -0,0 +1,49 @@
+name: Improve existing content
+description: Make a suggestion to improve the content in an existing article
+title: '[What I really want the title to be]'
+labels:
+  - content
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **HUBBERS!! THIS IS GITHUB/DOCS OPEN SOURCE REPO. OPEN AN ISSUE IN GITHUB/DOCS-CONTENT INSTEAD.**
+
+        * For questions, ask in [Discussions](https://github.com/github/docs/discussions)
```

Comment:
Can you go through and add a `.` to all of the complete sentences? We use periods at the end of any list item that's a complete sentence (like these ones), there's some more details in the [style guide](https://slack.com/intl/en-nl/help/articles/204475027-Deactivate-a-members-account) ✨

Reply:
I am going to add `.` to the list points 🐬 Should I open another PR or can I just make the changes in this PR?